src/shortcuts/ShortcutManager.js

```diff
@@ -334,7 +334,10 @@ class ShortcutManager {
                 isSet = context.getAttributeIsSet(parentAttribute);
                 isSettable = Lang.isUndefined(voa.isSettable) ? false : (voa.isSettable === "true");
                 if (isSettable) { // if is settable and not set, then we want to include the shortcut
-                    if (context.getAttributeIsSetByLabel(parentAttribute)) return false; // If attribute was set by label then we should not include the shortcut
+                    if (typeof context.getAttributeIsSetByLabel === 'function') {

```

Comment:
Not sure if this is better, but why not just use `if (context.getAttributeIsSetByLabel)` instead of checking the typeof?